pyproject.toml

```diff
@@ -67,6 +67,9 @@ joblib = "*"
 openhands-aci = "^0.1.2"
 python-socketio = "^5.11.4"
 redis = "^5.2.0"
+fuzzywuzzy = "^0.18.0"
+rouge = "^1.0.1"
+python-levenshtein = "^0.26.1"
```

Comment:
These should not be incorporated into the main dependencies to prevent dependency bloat. We can have it as dependencies only for TestGenEval.

Reply:
Addressed this in updated version